vpr/src/route/rr_graph.cpp

```diff
@@ -507,6 +507,8 @@ static void build_rr_graph(const t_graph_type graph_type,
     /* get maximum number of pins across all blocks */
     int max_pins = types[0].num_pins;
     for (const auto& type : types) {
+        VTR_LOG("%s\n", type.name);
```

Comment:
It seems that this log message was used for debugging. Should remove it once finished debugging.